sgt-back-end/index.js

```diff
@@ -0,0 +1,134 @@
+// eslint-disable-next-line no-console
+const pg = require('pg');
+const express = require('express');
+const app = express();
+
+const middleWare = express.json();
+app.use(middleWare);
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:lfz@localhost/studentGradeTable'
+});
+
+app.get('/api/grades', (req, res) => {
+  const sql = `
+  select *
+  from "grades"
+  `;
+  db.query(sql)
+    .then(result => res.status(200).json(result.rows))
+    .catch(err => {
+      console.error(err);
+      res.status(500).json({ error: 'An unexpected error occured' });
+    });
+});
+
+app.post('/api/grades', (req, res) => {
+  const id = req.body;
+  if (!Number.isInteger(id.grade) || id.grade <= 0) {
+    return res.status(400).json({
+      error: 'invalid grade'
+    });
+  }
+
+  const sql = `
+ insert into "grades" ("name","course","grade")
+ values ($1, $2, $3)
+ returning *
+ `;
+  const values = [id.name, id.course, id.grade];
+
+  db.query(sql, values)
+    .then(result =>
+      res.status(201).json(result.rows))
```

Comment:
The client should receive the row and not an array containing the row. A waiter pours your glass of wine for you.